app/src/main/java/com/x8bit/bitwarden/data/autofill/accessibility/BitwardenAccessibilityService.kt

```diff
@@ -22,8 +22,7 @@ class BitwardenAccessibilityService : AccessibilityService() {
     lateinit var processor: BitwardenAccessibilityProcessor
 
     override fun onAccessibilityEvent(event: AccessibilityEvent) {
-        if (rootInActiveWindow?.packageName != event.packageName) return
```

Comment:
@fedemkr & @mpbw2 

Do you know how important the package name check is?
I have not seen it do anything in my testing but this check existed in the Maui app and I wanted to make sure it was OK to remove.

Reply:
So I left the check in but reduced our calls to `rootInActiveWindow` down to one. This should still have a positive impact without losing the package check.